src/datasources/track-api.js

```diff
@@ -1,33 +1,33 @@
-const { RESTDataSource } = require('apollo-datasource-rest');
+const { RESTDataSource } = require("apollo-datasource-rest");
 
 class TrackAPI extends RESTDataSource {
   constructor() {
     super();
     // the Catstronauts catalog is hosted on this server
-    this.baseURL = 'https://odyssey-lift-off-rest-api.herokuapp.com/';
+    this.baseURL = "https://odyssey-lift-off-rest-api.herokuapp.com/";
```

Comment:
I believe we need to get rid of this `constructor` method as well and simply do 

`baseUrl = "..."`

The caching didn't work properly for me if I left the constructor method.